src/store/songs/actions.js

```diff
@@ -6,6 +7,7 @@ export const changeVote = song => (dispatch, getState, getFirebase) =>
     const votes = song.votes || 0
     const newVotes = votes + value
     try {
+      writeVoteToStorage(song.id)
       await firebase.update(`/songs/${song.id}`, { votes: newVotes })
     } catch (error) {
```

Comment:
we will want to undo the write if there is a failure here, but that can be done in the branch that allows the user to undo votes.